doc/Installation.md

```diff
@@ -59,6 +122,23 @@ container.
 
 For other make targets, inspect the Makefile.
 
+### Further notes:
+
+If the steps above didn't work try the next:
+
+  > make
+
+  > make install_qmstr_client
```

Comment:
This should be `make install_qmstr_client_gopath` in order to avoid needing sudo. The target `install_qmstr_client` is meant to be called when building container images.

Reply:
I have changed it, please have a look.